test/WebSites/InlineConstraintsWebSite/Constraints/IsbnDigitScheme10Constraint.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Routing;
+
+namespace InlineConstraintsWebSite.Constraints
+{
+    public class IsbnDigitScheme10Constraint : IRouteConstraint
+    {
+        private readonly bool _turnOn;
+
+        public IsbnDigitScheme10Constraint(bool turnOn)
```

Comment:
The reason why I have two different constraints is for testing updating scenario for constraintmap. And, I also didn't want to update an existing constraint since this WebSite is used for the rest of the functional tests too. 
In this case, turnOn isn't useful, but.. this is just to show how users can pass a parameter to the constructor.

Let me know if you think we don't have to test the update scenario or...we don't have to show an example for passing a parameter to the constraint's constructor.